src/core/cache.ts

```diff
@@ -24,6 +24,12 @@ interface CacheEntries {
   lobbyIds: string[];
   onDemandChannels: string[];
   quoiFeurChannels: string[];
+  cookieHunterChannels: string[];
+  currentHuntMessageId: string;
+  cookieHunterDailyCount: Record<string, number>;
+  cookieHunterDailyLogChannels: string[];
```

Comment:
what about adding a bit of jsdoc on top of some of these entries? 

it may not be obvious what we are storing here without looking at the code

is `cookieHunterDailyCount` a discord user id <> score record? 
what is `cookieHunterDailyLogChannels` ?